web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-troubleshooting.adoc

```diff
@@ -0,0 +1,166 @@
+. Add or uncomment the following line:
++
+[source,xml,linenums]
+----
+<AsyncLogger name="org.mule.service.http.impl.service.HttpMessageLogger" level="DEBUG" />
+----
++
+. Save all your application changes.
+.  Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+. Navigate to the *Console* view to read the HTTP messages in the Mule's logger, for example:
++
+
+[source,plain-text]
+----
+POST /essentials/delta HTTP/1.1
+soapaction:
+Host: ilt.mulesoft-training.com
+User-Agent: AHC/1.0
+Connection: keep-alive
+Accept: */*
+Content-Type: text/xml; charset=UTF-8
+Transfer-Encoding: chunked
+
+b1
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><ns0:listAllFlights xmlns:ns0="http://soap.training.mulesoft.com/"/></soap:Body></soap:Envelope>
+
+DEBUG 2021-04-06 15:04:23,058 [[wsc-base].wsc-dispatcher.11 SelectorRunner] [processor: ; event: ] org.mule.service.http.impl.service.HttpMessageLogger.wsc-dispatcher: REQUESTER
+0
+
+
+DEBUG 2021-04-06 15:04:23,319 [[wsc-base].wsc-dispatcher.11 SelectorRunner] [processor: ; event: ] org.mule.service.http.impl.service.HttpMessageLogger.wsc-dispatcher: REQUESTER
+HTTP/1.1 200
+Date: Tue, 06 Apr 2021 18:04:23 GMT
+Content-Type: text/xml; charset=UTF-8
+Transfer-Encoding: chunked
+Connection: keep-alive
+MULE_ENCODING: UTF-8
+Strict-Transport-Security: max-age=31536000; includeSubdomains;
+X-Content-Type-Options: 1
+X-Frame-Options: SAMEORIGIN
+X-XSS-Protection: 1; mode=block
+
+bfe
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><ns2:listAllFlightsResponse xmlns:ns2="http://soap.training.mulesoft.com/">...
+----
+
+== Troubleshoot Access Attachments
+
+Some servers can respond with MTOM messages. These messages separate their data into parts which enables different content types of data to coexist in the same HTTP message.
+
+MTOM is in charge of sending binary data in a separate part and referencing the part from the corresponding field within the SOAP envelope.
+Web Service Consumer users interact with these messages (multipart) only with the MTOM configuration. This is not to say that MTOM, XOP, and MIME or multipart are the same.
+
+Refer to the following documentation for further information:
+
+* https://www.ietf.org/rfc/rfc2387.txt[RFC2387 - The MIME Multipart/Related Content-type]
+* https://www.w3.org/TR/2005/REC-xop10-20050125/[XML-binary Optimized Packaging (XOP) W3C Recommendation]
+* https://www.w3.org/TR/soap12-mtom/[SOAP Message Transmission Optimization Mechanism (MTOM) W3C Recommendation]
+
+
+Since version 1.6.4, Web Service Consumer Connector enables you to configure MTOM for request messages, that also automatically manages MTOM responses regardless of the MTOM setting in the Web Service Consumer global configuration. Thereby, incoming messages with MTOM are treated as MTOM enabled regardless of whether MTOM is configured in the Web Service Consumer global configuration.
```

Comment:
GLOBALLY avoid "since" because of its multiple meanings. Also, doesn't v1.6.4 refer to the Consumer and not the Connector?
```suggestion
Beginning with version 1.6.4, Web Service Consumer Connector enables you to configure MTOM for request messages that also automatically manage MTOM responses, regardless of the MTOM setting in the Web Service Consumer global configuration. Incoming messages with MTOM are treated as MTOM-enabled regardless of whether MTOM is configured in the Web Service Consumer global configuration.
```
If this is so...why do we need all the previous text about having to configure MTOM in WSC?
I think you want to revisit this text section, avoiding all the MTOM background and saying something like before 1.6.4 of WSC, you had to enable MTOM in WSC config, but now you....